tools/net-changelog-gen-mgmt/Azure.SDK.Management.ChangelogGen/Azure.SDK.Management.ChangelogGen.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Why did you convert to .NET 7? We have been trying to keep our tooling on LTS which is .NET 6.

Reply:
just know we are trying to keep on .net 6. will revert it back to .net6. thanks.